zerver/models/streams.py

```diff
@@ -170,6 +173,13 @@ class Stream(models.Model):
             allow_everyone_group=False,
             default_group_name="stream_creator_or_nobody",
         ),
+        "can_move_messages_within_channel_group": GroupPermissionSetting(
+            require_system_group=False,
+            allow_internet_group=False,
+            allow_nobody_group=True,
+            allow_everyone_group=True,
+            default_group_name=SystemGroups.NOBODY,
+        ),
```

Comment:
@sahil839 what do you think about making `require_system_group` and `allow_internet_group` optional parameters to `GroupPermissionSetting` with a default of `False`? Between the two of those, there's exactly two settings where one of them is `True`, and I feel like it'd reduce clutter to not repeat them in every setting.

Reply:
Yes, we can do that. Will open a PR.